meta-phosphor/common/recipes-kernel/linux/linux-obmc.bb

```diff
@@ -0,0 +1,26 @@
+DESCRIPTION = "Linux kernel for OpenBMC"
+SECTION = "kernel"
+LICENSE = "GPLv2"
+
+KBRANCH ?= "dev"
+
+inherit kernel
+require recipes-kernel/linux/linux-yocto.inc
+
+SRC_URI = "git://github.com/openbmc/linux;protocol=git;branch=${KBRANCH}"
+SRC_URI += "file://obmc-bsp.scc \
+            file://obmc-bsp.cfg \
+            file://obmc-bsp-user-config.cfg \
+            file://obmc-bsp-user-patches.scc \
+           "
+
+LINUX_VERSION ?= "4.3-rc1"
```

Comment:
No, I just saw this somewhere on the openbmc/linux page and copied it.  The checkout that is done is just the head of the dev branch.

The package version is derived from this variable.  We don’t have new/old versions of packages yet so this is sort of a no-op at the moment.  I will make it reflect 4.2.

Thx - Brad

> On Sep 30, 2015, at 8:00 PM, Joel Stanley <notifications@github.com> wrote:
> 
> In meta-phosphor/common/recipes-kernel/linux/linux-obmc.bb:
> 
> > +SECTION = "kernel"
> > +LICENSE = "GPLv2"
> > +
> > +KBRANCH ?= "dev"
> > +
> > +inherit kernel
> > +require recipes-kernel/linux/linux-yocto.inc
> > +
> > +SRC_URI = "git://github.com/openbmc/linux;protocol=git;branch=${KBRANCH}"
> > +SRC_URI += "file://obmc-bsp.scc \
> > +            file://obmc-bsp.cfg \
> > +            file://obmc-bsp-user-config.cfg \
> > +            file://obmc-bsp-user-patches.scc \
> > +           "
> > +
> > +LINUX_VERSION ?= "4.3-rc1"
> 
> Will this attempt to check out 4.3-rc1? The development we're doing is based on 4.2.
> 
> —
> Reply to this email directly or view it on GitHub.
>